MekHQ/src/mekhq/campaign/randomEvents/personalities/enums/Aggression.java

```diff
@@ -0,0 +1,179 @@
+ */
+package mekhq.campaign.randomEvents.personalities.enums;
+
+import megamek.common.enums.Gender;
+import megamek.logging.MMLogger;
+import mekhq.campaign.personnel.Person;
+
+import static mekhq.campaign.personnel.enums.GenderDescriptors.HE_SHE_THEY;
+import static mekhq.campaign.personnel.enums.GenderDescriptors.HIM_HER_THEM;
+import static mekhq.campaign.personnel.enums.GenderDescriptors.HIS_HER_THEIR;
+import static mekhq.utilities.MHQInternationalization.getFormattedTextAt;
+
+public enum Aggression {
+    // region Enum Declarations
+    NONE(false, false),
+    AGGRESSIVE(false, false),
+    ASSERTIVE(true, false),
+    BELLIGERENT(false, false),
+    BOLD(true, false),
+    BRASH(false, false),
+    CONFIDENT(true, false),
+    COURAGEOUS(true, false),
+    DARING(true, false),
+    DECISIVE(true, false),
+    DETERMINED(true, false),
+    DOMINEERING(false, false),
+    FEARLESS(true, false),
+    HOSTILE(false, false),
+    HOT_HEADED(false, false),
+    IMPETUOUS(false, false),
+    IMPULSIVE(false, false),
+    INFLEXIBLE(false, false),
+    INTREPID(true, false),
+    OVERBEARING(false, false),
+    RECKLESS(false, false),
+    RESOLUTE(true, false),
+    STUBBORN(false, false),
+    TENACIOUS(true, false),
+    VIGILANT(true, false),
+    // Major Traits should always be last
+    BLOODTHIRSTY(false, true),
+    DIPLOMATIC(true, true),
+    MURDEROUS(false, true),
+    PACIFISTIC(true, true),
+    SAVAGE(false, true),
+    SADISTIC(false, true);
+
+    // endregion Enum Declarations
+
+    // region Variable Declarations
+    private final boolean isPositive;
+    private final boolean isMajor;
+    // endregion Variable Declarations
+
+    // region Constructors
+    Aggression(boolean isPositive, boolean isMajor) {
+        this.isPositive = isPositive;
+        this.isMajor = isMajor;
+    }
+    // endregion Constructors
+
+    final private String RESOURCE_BUNDLE = "mekhq.resources." + getClass().getSimpleName();
+
+    /**
+     * Retrieves the label associated with the current enumeration value.
+     *
+     * <p>The label is determined based on the resource bundle for the application,
+     * utilizing the enum name combined with a specific key suffix to fetch the
+     * relevant localized string.</p>
+     *
+     * @return the localized label string corresponding to the enumeration value.
+     */
+    // region Getters
+    public String getLabel() {
+        final String RESOURCE_KEY = name() + ".label";
+
+        return getFormattedTextAt(RESOURCE_BUNDLE, RESOURCE_KEY);
+    }
+
+    /**
+     * Retrieves the description of the current enumeration value.
+     *
+     * <p>The label is determined based on the resource bundle for the application,
+     * utilizing the enum name combined with a specific key suffix to fetch the
+     * relevant localized string.</p>
+     *
+     * @return the description associated with this enumeration value
+     */
+    public String getDescription(Person person) {
```

Comment:
Could it make sense to pass in the index, first name, and gender instead of a whole person to reduce the coupling with that class? Doesn't seem like a major concern here but the fewer things you have to mock in tests the better usually.